documentation/modules/ROOT/pages/_partials/watching-logs.adoc

```diff
@@ -1,7 +1,8 @@
 [kube-ns='knativetutorial']
 [kube-svc='']
 
-Since a Cron job source is used in this section of the tutorial, it would emit events every minute. We can watch the logs of the service to see the messages delivered.
+In the eventing related subsections of this tutorial, event sources are configured to emit events every minute with a `CronJobSource` or with a `ContainerSource`.  
```

Comment:
Generalized this message so that it applies to cron job sources (source to service and source to subscriber sections) as well as container sources (broker+trigger section).

Note that the container event source is also configured to send the same message as the cron job source with the same frequency; so the docs here match.